### ISSUE TITLE
Allow users to select time of arrival before booking

### ISSUE THREAD

**Issue by road2nowhere** [where2help/creative#6]
After slack and Trello, now also on Github - sorry for the spamming;)

I think we need to give users the possibility to specify their time of arrival, otherwise the experience might easily get frustrating and people might stop using it. Currently the system automatically assigns a time. If the user is not available at that time, there's no way to specify another time, you have to try randomly. I imagine a simple drop down in the detail page with times in a 2-hour interval (9:00, 11:00,...) and already taken ones greyed out or just not visible. Can we add the time of arrival to the detail page?

**Comment by klappradla** [comment]
As the other devs said, we should wait for feedback from the beta users before adding a feature that high in complexity. It is not within the scope of an MVP, so it would be better to postphone this discussion until we have the MVP.

**Comment by road2nowhere** [comment]
Yes, if we can change it after beta if needed then that's a good idea. (Would have made sense to anticipate, if it was less costly in terms of development now as compared to later, but this doesn't seem to be the case.) Roger is also informed and fine with this.
